Documentation/bpf.rst

```diff
@@ -4,180 +4,185 @@
 BPF and XDP Reference Guide
 ***************************
 
-.. note:: This purely serves as a developer's guide and is explicitly not a
-          requirement for Cilium users to read.
-
-eBPF is a minimal, highly flexible and efficient "virtual machine"-like construct
-in the Linux kernel which is used in a number of subsystems, most prominently
+.. note:: This documentation section is targeted at developers and users who
+          want want to understand BPF and XDP in great technical depth. While
+          reading this reference guide may help broaden yur understanding of
+          Cilium, it is not a requirement to use Cilium. Please refer to the
+          :ref:`_gs_guide` and :ref:`_arch_guide` for a higher level
+          introduction.
+
```

Comment:
If you want the links to be rendered you need to drop the first underscore,
o s/_gs_guide/gs_guide
o s/_arch_guide/arch_guide